env.sh

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 
-export PROJECT="ssv-apigee-asm-demo"
+export PROJECT="$(gcloud config get project)"
```

Comment:
agreed but at least it errors out, no? 
otherwise let's change it to `my-project` or something generic doesn't actually exist or contains your personal prefix? Same for the Hostname I guess. The proxy is still up so error messages on tokens might be confusing/misleading.

Reply:
yes - this was a wrong commit from my side, I have updated the env.sh to default values